openapi/openapi-collections.ytt.yaml

```diff
@@ -127,6 +127,22 @@ paths:
           schema:
             type: integer
       responses: #@ response(type("boolean"))
+  
+  /collections/{collection_name}/exists:
+    get:
+      tags:
+        - collections
+      summary: Check if collection exists
+      description: Check if collection exists
+      operationId: collection_exists
+      parameters:
+        - name: collection_name
+          in: path
+          description: Name of the collection to check
+          required: true
+          schema:
+            type: string
+      responses: #@ response(type("boolean"))
```

Comment:
This makes the response non-extensible, let's use an object instead. Same way as in the proto spec

Reply:
Sure

Reply:
the response (if 200) is in this format: 
```json
{
    "result": true
}
```

Do you want me to return something else? Or just update the type in the .ytt schema?

Reply:
more like
```
{
    "result": {
         "exists": true
    }
}
```

Reply:
ah, cool!